hips/hip-9999.md

```diff
@@ -0,0 +1,71 @@
+---
+hip: 9999
+title: "Helm Triage Maintainers"
+authors: [ "Matthew Fisher <matt.fisher@microsoft.com>" ]
+created: "2021-07-06"
+type: "process"
+status: "draft"
+---
+
+## Abstract
+
+The existing Helm Project Maintainers have identified the need for a new "triage" role. This document lists out the responsibilities of this new role.
```

Comment:
If I understand this right, this is for helm/helm only. So, we have Helm Org -> Helm sub-project (helm/helm) -> triage role. This is not for other sub-projects. Is that right?

Since this is a sub-group, I wonder if the org maintainers need to weight in... per the governance "Deciding what sub-groups are part of the Helm project" is an org maintainer role. Thoughts?

Reply:
> This is not for other sub-projects. Is that right?

Yes, that is correct.

> I wonder if the org maintainers need to weight in... per the governance "Deciding what sub-groups are part of the Helm project" is an org maintainer role. Thoughts?

Good question. I figured that project maintainers have full autonomy over how they choose to govern their own projects. The project maintainers are still responsible for discussing org-wide concerns and report to the org maintainers. Project maintainers can create sub-sub groups that report back to them, but project maintainers cannot create another sub-group under the Helm org.

That being said, this document does mention that triage maintainers have the same voting rights as project maintainers, as they can vote on governance-related topics. Perhaps this terminology should be restricted down to voting rights with regards to the helm project (i.e. voting for or against new core maintainers, not voting for/against new org maintainers).

What are your thoughts here?

Reply:
Based on https://github.com/helm/community/pull/199#discussion_r683680226 we will have to revoke this capability until tag protection rules have been implemented.

Reply:
After further discussion with other org maintainers it was decided that triage maintainers should not be granted voting rights. Nominations are fine though. That way the separation between triage maintainers and core maintainers are clear: core maintainers are responsible for the project and serve as mentors, while triage maintainers can help with the day-to-day but cannot influence the project's direction through governance UNTIL they become core maintainers.

Reply:
Sounds good to me. I think that clarification will end up being the real distinguishing mark, and give the whole thing a little more of a "ladder" feel.